<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>634</width>
    <height>389</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resources.qrc">
    <normaloff>:/images/src/icon.ico</normaloff>:/images/src/icon.ico</iconset>
  </property>
  <property name="locale">
   <locale language="English" country="UnitedStates"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="timeOptionsWidget" native="true">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>100</y>
      <width>131</width>
      <height>251</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="intervalOptionsWidget" native="true">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>210</y>
      <width>411</width>
      <height>71</height>
     </rect>
    </property>
    <widget class="QPushButton" name="s30Button">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>61</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>30s</string>
     </property>
    </widget>
    <widget class="QPushButton" name="m1Button">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>10</y>
       <width>61</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>1m</string>
     </property>
    </widget>
    <widget class="QPushButton" name="m5Button">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>10</y>
       <width>61</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>5m</string>
     </property>
    </widget>
    <widget class="QPushButton" name="m30Button">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>10</y>
       <width>61</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>30m</string>
     </property>
    </widget>
    <widget class="QPushButton" name="h1Button">
     <property name="geometry">
      <rect>
       <x>330</x>
       <y>10</y>
       <width>61</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>1h</string>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="startButton">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>300</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background-color: #3d3d3d;
    color: white; 
    border: none;
    padding: 10px 20px;
    border-radius: 15px;
    font-size: 16px;
}

QPushButton:hover {
    background-color: #2980b9;
}

QPushButton:pressed {
    background-color: #1c5980; 
}</string>
    </property>
    <property name="text">
     <string>Start</string>
    </property>
   </widget>
   <widget class="QPushButton" name="clearButton">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>300</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background-color: rgb(0, 212, 190);
    color: white; 
    border: none;
    padding: 10px 20px;
    border-radius: 15px;
    font-size: 16px;
}

QPushButton:hover {
    background-color: #2980b9;
}

QPushButton:pressed {
    background-color: #1c5980; 
}</string>
    </property>
    <property name="text">
     <string>Reset</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="sleepModeCheckBox">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>310</y>
      <width>91</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>SleepMode?</string>
    </property>
   </widget>
   <widget class="QLCDNumber" name="labelCurrentTime">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>12</y>
      <width>201</width>
      <height>61</height>
     </rect>
    </property>
   </widget>
   <widget class="QLCDNumber" name="labelTimerValue">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>130</y>
      <width>271</width>
      <height>61</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="header">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>10</y>
      <width>651</width>
      <height>81</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="resources.qrc">:/images/src/header.png</pixmap>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>96</y>
      <width>21</width>
      <height>271</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="resources.qrc">:/images/src/line.png</pixmap>
    </property>
   </widget>
   <zorder>timeOptionsWidget</zorder>
   <zorder>intervalOptionsWidget</zorder>
   <zorder>startButton</zorder>
   <zorder>clearButton</zorder>
   <zorder>sleepModeCheckBox</zorder>
   <zorder>labelTimerValue</zorder>
   <zorder>header</zorder>
   <zorder>labelCurrentTime</zorder>
   <zorder>label</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>634</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
